shop-service/product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,23 @@
+import ProductService from "../productsService";
+import createResponse from "../helpers/createResponse";
+import { NOT_FOUND_ERROR, INTERNAL_SERVER_ERROR } from "../constants";
+
+export const getProductsById = async (event) => {
```

Comment:
please add types for parameters and return value

Reply:
done